stubs/pyinstaller/PyInstaller/building/splash.pyi

```diff
@@ -7,21 +7,23 @@ splash_requirements: list[str]
 # Referenced in https://pyinstaller.org/en/stable/spec-files.html#example-merge-spec-file
 # Not to be imported during runtime, but is the type reference for spec files which are executed as python code
 class Splash(Target):
-    image_file: str
-    full_tk: Incomplete
-    name: Incomplete
-    script_name: Incomplete
-    minify_script: Incomplete
-    max_img_size: Incomplete
-    text_pos: Incomplete
-    text_size: Incomplete
-    text_font: Incomplete
-    text_color: Incomplete
-    text_default: Incomplete
-    always_on_top: Incomplete
-    uses_tkinter: Incomplete
-    script: Incomplete
-    splash_requirements: Incomplete
+    image_file: StrPath
+    full_tk: bool
+    tcl_lib: str
+    tk_lib: str
+    name: StrPath
+    script_name: StrPath
```

Comment:
I didn't find anywhere where this being a `PathLike` would be problematic, although its support is likely incidental given the docstring.

But then can you also update the `script_name` param in `__init__` ?